Gopkg.toml

```diff
@@ -38,8 +38,8 @@ required = [
   branch = "master"
   name = "github.com/docker/distribution"
 
-[[constraint]]
-  branch = "master"
+[[override]]
+  branch = "hamasterspec"
   name = "github.com/giantswarm/apiextensions"
```

Comment:
Question is if we want to merge what we have in `apiextensions` and iterate from here or if we want to merge this PR and keep it on the feature branch for some time. Your call.

Reply:
I'll go and request review for the `apiextensions` PR :) I don't think it's dangerous to merge it.